frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualizationTask.tsx

```diff
@@ -39,7 +37,7 @@ interface TaskProps {
   task?: {
     data: TaskKind;
   };
-  status?: TaskStatus;
+  status: TaskStatus;
```

Comment:
Is this really defined when rendering a Pipeline?

Reply:
That's interesting because the `TaskComponent` doesn't "trust" the status and believe it could be null:

See `!!status` and `status?.reason`:

https://github.com/openshift/console/blob/9e51260acb31bd8042da93ff77b63ae7a066bf4a/frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualizationTask.tsx#L152-L161